lib/gs_schemas/src/coordinates.rs

```diff
@@ -14,6 +14,8 @@ pub const BLOCK_DIM: f32 = 0.5;
 /// Length of a side of a chunk in blocks
 pub const CHUNK_DIM: i32 = 32;
 /// Length of a side of a chunk in blocks
+pub const CHUNK_DIMF: f64 = CHUNK_DIM as f64;
```

Comment:
```suggestion
pub const CHUNK_DIMD: f64 = CHUNK_DIM as f64;
```
let's keep F for f32, D for f64.